Langs/README.md

```diff
@@ -0,0 +1,38 @@
+# Guidelines
+
+## Requirements
+
+- **Find Your Language Code Here:** https://www.autoitscript.com/autoit3/docs/appendix/OSLangCodes.htm
+- Always use the latest template (0000.lang)
+- Use hex code(number) of your language.
+- Please use your GitHub username as Translator in MetaData section.
+- All strings must be filled out to be accepted.
+- Due to space restrictions, translations do not have to be 1:1.
+- Please do not use slang, due to it being hard to judge/approve.
+- File must be UTF-8 encoded, use GitHub editor and it should take care of that.
+
+## Testing Translations Locally
+1. Go to folder: *%localappdata%\WhyNotWin11\langs*
+2. Find your language code
+3. Edit file and relaunch application
+
+## Submitting
+
+1. Open issue or pull request
+2. In the title please include language name and whether you add a new one or update existing
+
+## Review Process
+
+##### When New Translation
```

Comment:
I don't like the "when" usage here and below. Also I think it should say translations.
Also, why did you use 5 hashtags here and not three?

Reply:
I tried to make small headers. I don't know why I think that is good idea.